pkg/crc/constants/constants.go

```diff
@@ -113,12 +113,12 @@ var (
 	PasswdFilePath      = filepath.Join(MachineInstancesDir, InstanceDirName(), "passwd")
 )
 
-func InstanceDirName() string {
-	return InstanceName()
+func InstanceDirName(preset crcpreset.Preset) string {
```

Comment:
After this change, compilation will be broken for a few commits. Since it is a very mechanical change, doing everything in one go should be ok.